### DISCUSSION TITLE
What are positive and negative values indicate?

### DISCUSSION THREAD

**Discussion by utkarshpancholi** [simnibs/simnibs#154] [2. General]
If we map the simulation to the cortical middle surface and look at the normal component of the electric field to see that the pattern of the inflow and outflow changes, how to recognize inflow and outflow??

By considering red color as inflow and blue as outflow current ??

When we analyze central.E.normal file in subject overlays's folder in MATLAB, it shows below

peak fields
percentiles: 95 99 99.9
values: 0.00935 0.0436 0.0779 (in [V/m])
values (negative): -0.00931 -0.0315 -0.0589

focality
cutoffs: 50 75 (in % of 99.9 percentile)
values: 2.21e+03 864 (in square mm)
values (negative): 2.09e+03 728

What are positive and negative values indicate?

**Comment by oulap** [comment]
Yes, current flowing in to the cortex is color-coded as red and current flowing out of the cortex is color-coded as blue. Similarly positive values denote the current flowing in and negative values denote the current flowing out.
